client/src/Router.js

```diff
@@ -0,0 +1,18 @@
+import React from "react";
+import { Switch, Route, BrowserRouter } from "react-router-dom";
+
+import AddFlight from "./pages/addFlight";
+import Login from "./pages/login";
+import UpdateFlight from "./pages/updateFlight";
+
+const Router = () => (
+  <BrowserRouter>
+    <Switch>
+      <Route exact path="/" render={() => <Login />} />
```

Comment:
Having the login route use a render function prop is good, because that's where you can check whether they already have a logged-in token, and can redirect them right into /flights or whereever they should go 😀